ui/apps/platform/src/Containers/NetworkGraph/simulation/NetworkPolicySimulatorSidePanel.tsx

```diff
@@ -193,8 +202,8 @@ function NetworkPolicySimulatorSidePanel({
                         networkPolicyGenerationScope={networkPolicyGenerationScope}
                     />
                 </Flex>
-                <Stack hasGutter>
-                    <StackItem isFilled style={{ overflow: 'auto' }}>
+                <Flex direction={{ default: 'column' }}>
+                    <FlexItem style={{ overflow: 'auto' }}>
```

Comment:
Not strictly related to this PR, but the PF Stack styles include `height: 100%` which was causing overly large scroll areas after this change.